.sage/main.go

```diff
@@ -32,10 +32,9 @@ func DoubleGitVerifyNoDiff(ctx context.Context) error {
 }
 
 func Default(ctx context.Context) error {
-	sg.Deps(ctx, ConvcoCheck, GoLint, GoTest, FormatMarkdown, FormatYaml, BackstageValidate)
-	sg.SerialDeps(ctx, GoModTidy)
-	sg.SerialDeps(ctx, GoLicenses, GitVerifyNoDiff)
-	return nil
+	sg.Deps(ctx, DoubleGitVerifyNoDiff)
+	cmd := sggit.Command(ctx, "status", "--short")
```

Comment:
`DoubleGitVerifyNoDiff` wont fail even do a file were modified, but `git status --short` verifies that a file actually were changed.